### PR TITLE
feat: benchmark specification I2

### LINKED ISSUE #4: I2: Specification - Sensitivity and FDR against ground truth
Define a specification file for I2.

**Working definition:** Compare output of tool's RNA-seq-based site identification to "ground truth" from the orthogonal 3'-end sequencing data. 
 - Sensitivity = TP/(TP+FN)
 - FDR = FP/(TP+FP)

### PR DESCRIPTION
**Pull request by daneckaw**
Adds I2 specification (Compare output of tool's RNA-seq-based site identification to "ground truth" from the orthogonal 3'-end sequencing data). 
closes #4 

Adds files:
- benchmark specification: `challenges\identification\I2_benchmark\I2_specification.md`
- example output files: 
  - `challenges\identification\I2_benchmark\example_files\output1.json`

### CODE REVIEW

File: challenges/identification/I2_benchmark/I2_specification.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+1. Sensitivity = (TP/(TP+FN))
+2. FDR = (FP/(TP+FP))
+
+TP - true positives - PAS identified by the tool and present in the orthogonal dataset  
+FP - false positives - PAS identified by the tool and not present in the orthogonal dataset  
+FN - false negatives - PAS not identified by the tool but present in the orthogonal dataset
+
+The metrics should be computed for the following distance threshold between PAS identified by the tool in RNAseq dataset and PAS identified in orthogonal 3'end seq dataset:
+
+- 50 nt
```
Comment (mfansler) [inline challenges/identification/I2_benchmark/I2_specification.md:23]: I think 50 nt is good.

File: challenges/identification/I2_benchmark/I2_specification.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  | 1 | Sensitivity | N/A | Output 1 | Read file, parse JSON and extract attribute `sensitivity` | `vector` | N/A |
+  | 2 | FDR | N/A | Output 1 | Read file, parse JSON and extract attribute `FDR` | `vector` | N/A |
+  
+### Additional info metrics
+  
+  N/A
+
+[//]: # (References)
+  
+  [in1]: ./example_files/input1.bed
```
Comment (mfansler) [inline challenges/identification/I2_benchmark/I2_specification.md:78]: We should add these before merging. I can do this.

**Review by mfansler — CHANGES_REQUESTED** [review]
Looks great. Do others agree about 50 nt?

File: challenges/identification/I2_benchmark/I2_specification.md (hunk 1 above)
Comment (mrgazzara) [reply challenges/identification/I2_benchmark/I2_specification.md:23]: So we can generate a curve based on varied distances between the algorithm predicted PAS and 3'end seq dataset PAS (Like in supplementary figure 1 a in this recent paper [https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-021-00864-5/MediaObjects/41588_2021_864_MOESM1_ESM.pdf](url)) would doing from 0 to 100 nt in steps of 10 nt make sense?

Comment (mfansler) [reply challenges/identification/I2_benchmark/I2_specification.md:23]: Yes, I think that would be nice.

**Review by mfansler — APPROVED** [review]
Good to go!

**Pull request merged**